frontend/src/components/object/ObjectVersion.vue

```diff
@@ -142,24 +146,28 @@ watch( [props], () => {
               v-if="data.public || permissionStore.isObjectActionAllowed(
```

Comment:
just a thought..i think they can't even get to the file details page if they havent got READ permission. but if we re-use this component, then this is good.